web/pgadmin/authenticate/oauth2.py

```diff
@@ -151,6 +151,22 @@ def login(self, form):
                 current_app.logger.exception(error_msg)
                 return False, gettext(error_msg)
 
+        if self.oauth2_config[self.oauth2_current_client][
+           'OAUTH2_ADDITIONAL_CLAIMS']:
+            additinal_claims = self.oauth2_config[
+                self.oauth2_current_client
+            ]['OAUTH2_ADDITIONAL_CLAIMS']
+            allowed = self.__is_authorized_additional_claims(profile,
+                                                             additinal_claims)
+            if not allowed:
+                return_msg = "Your user it's not authorized to access" \
+                    " PgAdmin based on your claims in your ID Token. " \
+                    " Please contact your administrator."
+                audit_msg = f"The authenticated user {username} is not" \
+                    " authorized to access pgAdmin based on OAUTH2 config."
+                current_app.logger.warning(audit_msg)
+                return False, gettext(return_msg)
```

Comment:
gettext should be applied on the string not on a variable.

Reply:
hmm, sorry I don't understand, return_msg is a string variable, I just used the same code a few lines above

Reply:
i will return the return_msg without gettext